group-coordinator/src/test/java/org/apache/kafka/coordinator/group/consumer/ConsumerGroupTest.java

```diff
@@ -673,4 +675,32 @@ public void testValidateOffsetFetch() {
         // This should succeed.
         group.validateOffsetFetch("member-id", 0, Long.MAX_VALUE);
     }
+
+    @Test
+    public void testValidateGroupDelete() {
+        Uuid fooTopicId = Uuid.randomUuid();
+        ConsumerGroup consumerGroup = createConsumerGroup("foo");
+        assertEquals(ConsumerGroup.ConsumerGroupState.EMPTY, consumerGroup.state());
+        assertDoesNotThrow(() -> consumerGroup.validateGroupDelete());
```

Comment:
we can do `consumerGroup::validateGroupDelete` for this along with the other invocations in the test